components/time-picker/nz-time-picker.component.html

```diff
@@ -49,7 +50,9 @@
     [opened]="nzOpen"
     [nzClearText]="nzClearText"
     [nzAllowEmpty]="nzAllowEmpty"
+    (keyup.enter)="close()"
     [(ngModel)]="value">
   </nz-time-picker-panel>
+  <span tabindex=0 (focus)="close()" class="nz-tab-catching-span"></span>
```

Comment:
Why class "nz-tab-catching-span"?

Reply:
Also the syntax `tabindex=0` should be `tabindex="0"`

Reply:
> Why class "nz-tab-catching-span"?
So that it is easier to handle in the future (if there is need) plus it is a bit self-documenting.

> Also the syntax tabindex=0 should be tabindex="0"
Ok. I will fix that.